core/trino-main/src/main/java/io/trino/sql/planner/PlanFragmenter.java

```diff
@@ -119,6 +119,25 @@ public PlanFragmenter(
     }
 
     public SubPlan createSubPlans(Session session, Plan plan, boolean forceSingleNode, WarningCollector warningCollector)
+    {
+        return createSubPlans(
+                session,
+                new PlanFragmentIdAllocator(0),
+                plan,
+                new PartitioningScheme(Partitioning.create(SINGLE_DISTRIBUTION, ImmutableList.of()), plan.getRoot().getOutputSymbols()),
+                ImmutableList.of(),
+                forceSingleNode,
+                warningCollector);
+    }
+
+    public SubPlan createSubPlans(
```

Comment:
nit: if possible (here it is) for method overloads make shorter parameter list prefix of longer one.